doc/fertiScan-architecture.md

```diff
@@ -0,0 +1,175 @@
+# FertiScan DB Architecture
+
+This is the doc about the FertiScan Database Architecture
+
+``` mermaid
+
+---
+title: FertiScan DB Structure
+---
+erDiagram
+  user{
+    uuid id PK
+    string email
+    timestamp registration_date
+    timestamp updated_at
+  }
+  analysis {
+    uuid id PK
+    uuid user_id FK
+    boolean verified
+    uuid label_info fk
+    TIMESTAMP upload_date
+    TIMESTAMP updated_at
+    uuid fertilizer_id
+    uuid sample_id FK
+    uuid company_id FK
+    uuid manufacturer_id FK
+    uuid picture_id
+  }
+  fertilizer{
+    uuid id PK
+    string name "Unique"
+    string registration_number
+    timestamp upload_date
+    timestamp update_time
+    uuid latest_Analyses FK
+    uuid respo_id 
+    uuid fertilizer_information FK
```

Comment:
Not sure I get the difference between fertilizer_information and label_information. Besides the lot number what would be different?